### ISSUE TITLE
Improve "share video" link in "my videos"

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#528]
Currently we show the direct link and make it easily copyable. But if the video is only mounted in one realm, we likely only want to show that link, not the direct link. Or at the very least we want to make the direct link less prominent and suggest sharing the link "in the realm context".

**Comment by oas777** [comment]
Can this be closed in line with https://github.com/elan-ev/tobira/issues/743?

**Comment by oas777** [comment]
TBD: RSS Feed

**Comment by LukasKalbertodt** [comment]
> TBD: RSS Feed

Not sure what this means? We have a dedicated issue for RSS stuff: https://github.com/elan-ev/tobira/issues/791

**Comment by oas777** [comment]
That was just a reminder to include RSS in the "Share" section. Mentioned it in https://github.com/elan-ev/tobira/issues/791.

**Comment by oas777** [comment]
That was just a reminder to include RSS in the "Share" section. Mentioned it in https://github.com/elan-ev/tobira/issues/791.

**Comment by oas777** [comment]
Question: What "said page" are you referring to when saying: "There is a UI on said page that offers to share the direct link. And further down, there is a list of all pages that the videos is included in. Clicking on those brings the user to the "in page video", being able to share from there."

**Comment by oas777** [comment]
Like this?

![grafik](https://github.com/elan-ev/tobira/assets/1752342/d7041af1-5cc6-414e-88cd-ae5e12d7a38f)

**Comment by LukasKalbertodt** [comment]
Yes that's the page I am talking about.

**Comment by oas777** [comment]
Well, if you could add the text information to the page that would be good enough for me at the moment so that users have both direct links and page links. #dagraf?

**Comment by dagraf** [comment]
> Well, if you could add the text information to the page that would be good enough for me at the moment so that users have both direct links and page links. David?

I would suggest we move the "Share via direct link:", its textbox and explanations to the very bottom of the "Video details" page and alter the other text informations slightly. Maybe like this:

*_*Referencing pages**
This video is referenced on the following pages. If you want to share the video with its references, visit those pages and use the share options offered there.

- TITLE OF PAGE1
- TITLE OF PAGE2 

**Direct link to video without reference**
Textbox containing the link_

**Comment by oas777** [comment]
Makes sense to place it with "Referencing pages". But: Sharing the (direct) link is probably the most important feature on that page - which is now at the bottom. Couldn't we move "Referencing pages" up to "Share"?

**Comment by owi92** [comment]
We replaced the "share via direct link" thing with the share button in https://github.com/elan-ev/tobira/pull/1580. This (besides adding RSS in the share section) is of course only cosmetic, and doesn't solve this issue.
But it is relevant for the above suggestions in the sense that it no longer seems reasonable to tie the referencing pages directly to the share menu by moving either element up or down.

We can still add the text (i.e. "If you want to share the video with its references, visit those pages...") that David (and a part of Lukas' comment) suggested though. That would be the simplest solution, also considering that this "details" page *is still* meant to undergo significant renovations *at some point* in the hopefully not too distant future.

So **TLDR**: I would just add that text suggested by David, but not move any elements. That ok with everyone?

**Comment by oas777** [comment]
This text? "This video is referenced on the following pages. If you want to share the video with its references, visit those pages and use the share options offered there." Why do have to visit those pages when I could also right-click on the links? Sorry if I am confusing things.

**Comment by owi92** [comment]
Yes, that one.

> Why do have to visit those pages when I could also right-click on the links?

Basically just for the QR code I suppose. And the `start at` option maybe.

**Comment by oas777** [comment]
Oh yeah, the QR code - whose idea was that in the first place? "visit those pages" sounds similarly outdated. What about "This video is referenced on the following pages; right-click the series or the video link to share the video in context." Obviously I still dislike the "Share" option being so prominent at the top of the page with this text at its end, but if I understand you correctly, this will not be a forever solution.

**Comment by owi92** [comment]
Uh looks like I forgot to answer here.

> QR code - whose idea was that in the first place?

Wasn't that your idea? I don't remember.

> "visit those pages" sounds (...) outdated. 

I would argue the same could be said about telling people to use right-click to copy something.


Soo maybe something along the lines of "This video is referenced on the following pages; copy the link or use the share menu on the linked page to share it in context."
We could also add a small copy icon button next to the links.

**Comment by LukasKalbertodt** [comment]
This issue is just one manifestation of an underlying problem, which I now opened an issue for: #1768 
Lets at some point figure that one out properly, then we can continue here.
